### ISSUE TITLE
[BUG] Disabled plugins don't show up in the correct color

### ISSUE THREAD

**Issue by lucian929** [FrankHeijden/ServerUtils#39]
Server Version:
```
[00:28:48 INFO]: This server is running Pufferfish version git-Pufferfish-5 (MC: 1.18) (Implementing API version 1.18-R0.1-SNAPSHOT) (Git: 6804728 on HEAD)
```

ServerUtils Version: 3.1.2

ServerUtils messages.yml:
```
  plugins:
    header: ''
    prefix: '<#D24C9F>Plugins <#D24C9F>(<count><#D24C9F>): '
    format: '<#95A5A6><plugin>'
    format-disabled: '<white>௱ <#E63946><plugin>'
    separator: '<#95A5A6>, '
    last-separator: ' <#D24C9F>and '
    version: ' <#95A5A6>(<#25A244><version><#95A5A6>)'
    footer: '<white>'
```
Bukkit Plugins List:    
![image](https://user-images.githubusercontent.com/60053521/144770335-f3e0177d-356f-4da0-97a6-e102416777ea.png)
ServerUtils Plugins List: (missing one plugin because of the plugin hide feature.)
![image](https://user-images.githubusercontent.com/60053521/144770345-b919cf29-9d3f-4996-82c3-4aea715fc1b4.png)

**Comment by FrankHeijden** [comment]
Thanks for reporting, will be fixed in the next version :)

**Comment by FrankHeijden** [comment]
Fixed in 3.4.0 :)
